mxcube3/ui/actions/general.js

```diff
@@ -152,11 +176,15 @@ export function getInitialStatus() {
     const pchains = [
```

Comment:
We can add another issue for that, if you want - but as you were adding more calls yourself... Do as you prefer for now, but if it is not solved in this PR we can add another issue for this I think.